qa/runtests.sh

```diff
@@ -16,14 +16,17 @@ check_python_version() {
 
 echo "Create Virtualenv for Python deps ..."
 function prepare_venv() {
+        python3 -m pip install --user virtualenv
```

Comment:
it is not recommended to install the dependency here as actions already installs it ref: https://github.com/fabric8-analytics/fabric8-analytics-github-refresh-cronjob/blob/master/.github/workflows/ci-build.yml#L40